mobile/src/people/avatar.tsx

```diff
@@ -3,21 +3,20 @@ import { defaultAvatar } from './avatar-default';
 import { View, Image, StyleSheet, ViewStyle, ImageStyle, LayoutChangeEvent } from 'react-native';
 
 const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-        justifyContent: 'center',
-        alignItems: 'center'
-    },
     outerFrame: {
+        justifyContent: 'center',
+        alignItems: 'center',
         // Following attributes are default
         borderColor: '#2FAFCC',
-        borderWidth: 2
+        borderWidth: 2,
+        height: 200,
```

Comment:
Regarding height and width again. If we use onLayout method then we have its size. So no height, width or borderRadius should be specified in these styles. There is no such thing as default value for it.
If those are not specified yet (nulls during render), just do not display anything. @serjKim @demkalkov  can anybody see any issues with that logic?

Reply:
I don't see the ability to override the size, if we pass custom size through the props.